OvaryModelGenerator/PelvisModelGenerator.h

```diff
@@ -4,23 +4,24 @@
 #include <iostream>
 
 #include "vtkPolyData.h"
+#include "vtkSmartPointer.h"
 
 class PelvisModelGenerator{
 
-    public:
-        float intercristalDistance;
-        float pelvisScaleFactor;
-        vtkPolyData* pelvisOutputPolydata;
-
     public:
         PelvisModelGenerator();
         ~PelvisModelGenerator();
-        PelvisModelGenerator(const PelvisModelGenerator &other);
+        PelvisModelGenerator(const PelvisModelGenerator &other) = delete;
+        PelvisModelGenerator & operator=(const PelvisModelGenerator &other) = delete;
 
         void SetIntercristalDistancemm(float intercristalDistance); //sets intercristal distance of pelvis and scale factor
         void Generate();
         vtkPolyData* GetOutput();
 
+    private:
+        float m_IntercristalDistance;
+        float m_PelvisScaleFactor;
```

Comment:
The original name is absolutely fine. Using "m_" prefix for member variable is just a convention adopted by many people.